src/main/java/software/amazon/documentdb/jdbc/DocumentDbDatabaseMetaData.java

```diff
@@ -356,6 +356,11 @@ private void addColumnsForTable(final String columnNamePattern,
         }
     }
 
+    /**
+     * Warning: if any entry becomes used/unused, reflection of that change in the
+     *  ODBC driver is required as well. Files to be changed in the ODBC driver:
+     *  function GetColumn in src\odbc\src\query\column_metadata_query.cpp
```

Comment:
Let's not include any paths - this will because out of date. Just file name and preferably the class name or namespace.